code/game/objects/items/passport.dm

```diff
@@ -8,8 +8,11 @@
 	attack_verb = list("whipped")
 	hitsound = 'sound/weapons/towelwhip.ogg'
 	desc = "A set of identifying documents."
-	material = /decl/material/solid/cardboard
-	matter = list(/decl/material/solid/leather = MATTER_AMOUNT_REINFORCEMENT)
+	material = /decl/material/solid/paper
+	matter = list(
+		/decl/material/solid/leather = MATTER_AMOUNT_REINFORCEMENT,
+		/decl/material/solid/cardboard = MATTER_AMOUNT_REINFORCEMENT
```

Comment:
Does this need to eventually be replaced with cardstock? (I figure that's what this is supposed to be, since it's used for business cards and the like. Cardboard is made out of corrugated layers of cardstock.)